rx_signal_processing/dsp.cu

```diff
@@ -390,7 +380,18 @@ DSPCore::~DSPCore()
 /**
  * @brief      Allocates device memory for the RF samples and then copies them to device.
  *
- * @param[in]  total_samples  Total number of samples to copy.
+ * @param[in]  total_antennas         The total number of antennas.
+ * @param[in]  num_samples_needed     The number samples needed from each antenna ringbuffer.
+ * @param[in]  extra_samples          The number of extra samples needed for filter propagation.
+ * @param[in]  time_zero              The time the driver began collecting samples.
+ * @param[in]  start_time             The start time of the pulse sequence.
+ * @param[in]  ringbuffer_size        The ringbuffer size.
+ * @param[in]  first_stage_dm_rate    The first stage dm rate.
```

Comment:
'dm' should be expanded to 'decimation'

Reply:
Actually, the dm_rates are not being used anymore, you can remove them from argument list.